src/hooks/useSignUpForm.tsx

```diff
@@ -0,0 +1,57 @@
+import { ISignUp } from '../components/form/singUp';
+import { useState } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+const useSignUpForm = () => {
+  const [showPassword, setShowPassword] = useState(false);
+  const handleClickShowPassword = () => setShowPassword((show) => !show);
+
+  const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+  };
+
+  const {
+    handleSubmit,
+    register,
+    watch,
+    formState: { errors },
+  } = useForm<ISignUp>();
```

Comment:
![image](https://github.com/VladyslavGilyuk/chi-pet-project/assets/68470050/9f2401d8-7c0b-49ff-b914-df4e369e87f6)

don't see any diff. Please move 
`const [showPassword, setShowPassword] = useState(false);
  const handleClickShowPassword = () => setShowPassword((show) => !show);

  const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
    event.preventDefault();
  };

  const {
    handleSubmit,
    register,
    watch,
    formState: { errors },
  } = useForm<ISignUp>();`
 into useAuthForm(hooks folder) use it in useSignInForm and useSignUpForm. And move useSignInForm & useSignUpForm in to the folders with components

Reply:
useAuthForm created in hooks and used in useSignInForm and useSignUpForm hooks which are moved to components=>form=>signIn/signUp (right?).